test/unit/associations/belongs-to-many.test.js

```diff
@@ -11,7 +11,7 @@ const BelongsTo = require(__dirname + '/../../../lib/associations/belongs-to');
 const HasMany = require(__dirname + '/../../../lib/associations/has-many');
 const HasOne = require(__dirname + '/../../../lib/associations/has-one');
 const current = Support.sequelize;
-const Promise = current.Promise;
+var Promise = current.Promise;
```

Comment:
hah, but the test suite actually runs without error despite the jshint error. Maybe this is an issue with jshint?